_data/yousraHazem/yousraHazem.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Yousra Mohamed
+institution: University of Waterloo
+profile_pic: yousra.png
+quote: "Towards a more sustainable future"
```

Comment:
Remove double quotes from begin and end.

Reply:
@Mohammed-Shoaib  done. please check updated file.